ui/app/mainui/AppMain.qml

```diff
@@ -954,11 +962,19 @@ Item {
             appMain.rootStore.mainModuleInst.setActiveSectionBySectionType(sectionType)
 
             if (sectionType === Constants.appSection.profile) {
-                profileLoader.settingsSubsection = subsection
-                profileLoader.settingsSubSubsection = subSubsection;
+                profileLoader.settingsSubsection = subsection || Constants.settingsSubsection.profile
```

Comment:
Coul you explain why this `|| Constants.settingsSubsection.profile`?

Reply:
That's the default Settings subsection (Profile), if not provided